packages/sanity/src/core/releases/components/Table/Table.tsx

```diff
@@ -190,12 +190,13 @@ const TableInner = <TableData, AdditionalRowTableData>({
   )
 }
 
-export const Table = <TableData, AdditionalRowTableData = undefined>(
-  props: TableProps<TableData, AdditionalRowTableData>,
-) => {
+export const Table = <TableData, AdditionalRowTableData = undefined>({
+  defaultSort,
+  ...props
+}: TableProps<TableData, AdditionalRowTableData> & {defaultSort?: TableSort}) => {
```

Comment:
Any reason defaultSort couldn't be part of TableProps? (only curious)

Reply:
`defaultSort` is only relevant now for the prop passed to the `TableProvider`, whereas `TableProps` is used also in `TableInner` (where defaultSort is not used)